codec/av1.md

```diff
@@ -106,14 +104,30 @@ The `Block` timestamp is translated from the __[PresentationTime]__ without the
 
 # Segment Restrictions
 
-Matroska doesn't allow dynamic changes within a codec for the whole `Segment`. The parameters that should not change for a video `Track` are the dimensions and the `CodecPrivate`. 
+AV1 stored in Matroska restricts the allowed variations among the `Sequence Header OBUs` contained in the `CodecPrivate` and also in-band in the `Blocks`. The changes are restricted to the changes that are allowed for the `Sequence Header OBUs` of a `CVS`, i.e. the contents of the `Sequence Header OBUs` MUST be bit-identical each time a `Sequence Header OBU` appears except for the contents of __[operating_parameters_info]__. Furthermore the dimensions of all output frames MUST be equal.
+
+

```

Comment:
Nit: please only one empty line.

Reply:
Thanks for taking a look, but this is directly in front of a header on the same level as e.g. "Block Data" or "Segment Restrictions" and there are two empty lines in front of these, too. I just imitated the style already in use.

Reply:
I see, then please ignore my comment. 
(Yet it’s not the case before `# Encryption` just below, which actually gave me the hint ;-)

Reply:
Ok, this is an oversight. Should I just add a new commit or amend the last one (rewriting history is after all frowned upon when one has already made something public and I honestly don't know what will happen with this PR when I replace one of the commits it contains with another one. Is the new/updated commit then automatically part of this PR?)

Reply:
Oh, in my personal opinion the History will stay exactly the same either way, but possibly my colleagues are more orthodox on this than I am. Let’s wait and see.